### ISSUE TITLE
Setup not working

### ISSUE THREAD

**Issue by pekeler** [aaronn/slack-gpt#1]
Hi, I've been trying to follow the setup instructions but they seem wrong, or at least in the wrong order.

**Comment by CoreyH** [comment]
Indeed, the order of the Slack app setup is incorrect, but if you do all the steps, it will eventually work.

**Comment by kellemar** [comment]
Hey didn't comment on this. Yeah the order is a bit off since i followed directly from the main repo.
